app/controllers/admin_controller.rb

```diff
@@ -151,7 +151,7 @@ def mark_comment_spam
     else
       flash[:error] = 'Only moderators can moderate comments.'
     end
-    redirect_to '/dashboard' + '?_=' + Time.now.to_i.to_s
+    redirect_back(fallback_location: root_path)
```

Comment:
Will this bust the cache the way the timestamp code does?

Reply:
It will redirect to the page from where the request is generated so - If I approve a comment from a research note, then the page will just reload(redirecting to requesting page which is note only). As the behaviour is reload so we don't need to bust cache explicitly (I don't faced cache issue while testing). In case, redirect_back wasn't able to redirect to the page from where request is generated then it will redirect to the root_path which is dashboard

Fallback demo:
![fallback_location](https://user-images.githubusercontent.com/20878070/52309680-e98b6180-29c6-11e9-979d-7226cee5d55c.gif)